Assets/Dev/Si Wai/Statistics/PermanentStatistic.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+using System;
+
+[Serializable]
+public class PermanentStatistic : BaseStatistic
+{
+    public PermanentStatistic() {}
+    public float PermanentBaseMulitpliers() => baseMultipliers.Any() ? baseMultipliers.Sum(pair => pair.Value) : 1;
+    public float PermanentFinalMulitpliers() => finalMultipliers.Any() ? finalMultipliers.Sum(pair => pair.Value) : 1;
```

Comment:
Spelling mistake Mulitpliers > Multipliers

Reply:
Also btw. Its not that big of a problem but:
1. We are already using the Permanent class so we already know its permanent, so we dont have to have that in the method name.
2. I would move the method to the base class aswell. Since this functionality is used in both the classes. And now we are still doing it in 2 places again which makes no since if we have a baseclass